### ISSUE TITLE
Improvement: when don't go back to an empty 'inbox' after replying in-line to a comment

### ISSUE THREAD

**Issue by stephantual** [ruqqus/ruqqus#164]
Steps to reproduce:

- be logged in
- click the notification icon
- notice you have 10+ comments to reply to
- click reply on the first one
- press submit

You're now taken to the page where you posted the reply to.  

Expected behaviour: let us reply to multiple comments 'in line' on the notification page so that we don't have to keep pressing 'back', be shown an empty inbox (because it's assumed 'read'), having to click 'show all' and rinse, wash repeat.... 

TL;DR: In-line replies from the notification page that dont' redirect us, please! TX!

**Comment by jasonkhanlar** [comment]
Indeed! In addition to users that have JavaScript turned off/blocked/disabled, allow for AJAX-style commenting, where instead of redirecting to reload the page, the existing DOM is updated

**Comment by captainmeta4** [comment]
Fix coming in #211
